modal_deployment.py

```
@@ -23,6 +25,7 @@
 @app.function(image=image, concurrency_limit=10)
 async def entry(text: str, query: str):
     setup_logging(logging.ERROR)
+    logger = structlog.get_logger()
```

Ruff (0.8.2)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)